packages/ndla-ui/src/TopicIntroductionList/TopicShortcutItem.tsx

```diff
@@ -17,7 +17,7 @@ interface Props {
 const ShortcutItem = ({ shortcut: { id, tooltip, contentType, url, count } }: Props) => {
   const { t } = useTranslation();
   return (
-    <Tooltip id={`shortcut-tooltip-${id}`} tooltip={t('resource.shortcutsTooltip', { count })}>
+    <Tooltip tooltip={t('resource.shortcutsTooltip', { count })}>
```

Comment:
Noen spesiell grunn til at du har fjerna id?

Reply:
Tooltip trenger ikke lenger en id for å fungere. Radix skal håndtere dette for oss nå